rep-0153.rst

```diff
@@ -110,6 +118,10 @@ this document and includes the additional metadata fields.
 To make use of the new index file the new version of the Python package will
 update the default URL to point to the v4 file.
 
+A new enough ``rosdistro`` version is necessary to access the later added key
+``python_version``.
+Older versions of ``rosdistro`` will simply ignore the key and not expose it.
```

Comment:
Can we decide now that the assuming this REP is accepted that the next rosdistro release will include this key, and thus specify that you'll need a version of rosdistro strictly newer than `0.7.4`? I think that having this will make these fuzzier changes to the specification much easier to deal with.

Reply:
I don't think that is something we have to specify. The next `bloom` release should probably have a version dependency on the to-be-released `rosdistro` version. But that is nothing I would state here.

The implicit assumption that users should use an up-to-date version of `bloom` is already currently the case and not checked / enforced anywhere.

Reply:
I don't think we have to specify it. I think it would be nice to because "new enough" is relative and in this case we are pretty confident what the "new enough" version is likely to be called, 0.8.0 or 0.7.5.

I'm not thinking about bloom, which will bump its dependency requirement, but about other users of the rosdistro API and index.

Reply:
Ok, got it. Yes, we can put specific version numbers in here. Done in 995d6d6db0af047c627a2bde1c11e8bbb94e7086.